catena4801_simple/catena4801_simple.ino

```diff
@@ -601,15 +601,15 @@ void startSendingUplink(void)
 
 	if(flagModBus == 0)
 		{
-		if (lastError == -4)
+		if (int(lastError) == -4)
```

Comment:
There are symbolic error constants... let's use them. The general rule is avoid use of magic numbers. Once you find the error code and substitute, you can go back to a normal compare `if (lastError == ERR_{whatever})` possibly with the official enum class name.

After we commit this cycle of changes, I'd like you to go back to the library and remove the use of the "legacy" header file, so you only use the pure c++ version. But that is not in this round of commits.

Reply:
Updated the error conditions with symbolic error constants.